excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/validation/SearchPartyValidatorTest.java

```diff
@@ -73,7 +76,7 @@ void setUp() {
     void shouldValidateAllFields() {
 
         // GIVEN
-        SearchPartyEntity searchPartyEntity1 = createPopulatedSearchPartyEntity();
+        SearchPartyEntity searchPartyEntity1 = createPopulatedSearchPartyEntityWithoutCollectionField();
```

Comment:
Please rename function and DisplayName e.g. "should validate all fields without collection field"